web/src/app/schedules/ScheduleCalendarOverrideDialog.js

```diff
@@ -0,0 +1,149 @@
+import React, { useContext, useState } from 'react'
+import { gql, useMutation } from '@apollo/client'
+import p from 'prop-types'
+import FormDialog from '../dialogs/FormDialog'
+import { DateTime } from 'luxon'
+import { fieldErrors, nonFieldErrors } from '../util/errutil'
+import useOverrideNotices from './useOverrideNotices'
+import { ScheduleCalendarOverrideForm } from './ScheduleCalendarOverrideForm'
+import { ScheduleCalendarContext } from './ScheduleDetails'
+
+export const variantDetails = {
+  add: {
+    title: 'Temporarily Add a User',
+    desc: 'This will add a new shift for the selected user, while the override is active. Existing shifts will remain unaffected.',
+    name: 'Add',
+    helperText: 'This will add a user to the schedule',
+  },
+  remove: {
+    title: 'Temporarily Remove a User',
+    desc: 'This will remove (or split/shorten) shifts belonging to the selected user, while the override is active.',
+    name: 'Remove',
+    helperText: 'This will remove a user from the schedule',
+  },
+  replace: {
+    title: 'Temporarily Replace a User',
+    desc: 'This will replace the selected user with another during any existing shifts, while the override is active. No new shifts will be created, only who is on-call will be changed.',
+    name: 'Replace',
+    helperText: 'This will replace a user from the schedule',
+  },
+  temp: {
+    title: 'Create a temporary schedule',
+    desc: 'Replace the entire schedule for a given period of time',
+    name: 'Temporary Schedule',
+    helperText: 'Replace the entire schedule for a given period of time',
+  },
+  choose: {
+    title: 'Choose an override action',
+    desc: 'This will create a temporary override to the existing schedule.',
+    name: 'Choose',
+    helperText: 'This will determine which override action you want to take',
+  },
+}
```

Comment:
I'd like to put this in more user-centric language:
```suggestion
export const variantDetails = {
  add: {
    title: 'Temporarily Add a User',
    desc: 'This will add a new shift for the selected user, while the override is active. Existing shifts will remain unaffected.',
    name: 'Additional Coverage',
    helperText: 'Add an additional on-call user for a specified time.',
  },
  remove: {
    title: 'Temporarily Remove a User',
    desc: 'This will remove (or split/shorten) shifts belonging to the selected user, while the override is active.',
    name: 'Remove Coverage',
    helperText: 'Remove one user's shifts for a specified time.',
  },
  replace: {
    title: 'Temporarily Replace a User',
    desc: 'This will replace the selected user with another during any existing shifts, while the override is active. No new shifts will be created, only who is on-call will be changed.',
    name: 'Cover Someone's Shifts',
    helperText: 'Have a user take over another's shifts for a specified time.',
  },
  temp: {
    title: 'Create a temporary schedule',
    desc: 'Replace the entire schedule for a given period of time',
    name: 'Temporary Schedule',
    helperText: 'Define a fixed shift-by-shift schedule to use for a specified time.',
  },
  choose: {
    title: 'Choose an override action',
    desc: 'This will create a temporary override to the existing schedule.',
    name: 'Choose',
    helperText: 'This will determine which override action you want to take',
  },
}
```